code/index.html

```diff
@@ -1,22 +1,123 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
 
-    <section class="">
+<head>
+  <meta charset="utf-8" />
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <style>
+    @import url('https://fonts.googleapis.com/css?family=Yantramanav&display=swap');
+  </style>
+  <style>
+    @import url('https://fonts.googleapis.com/css?family=Bungee&display=swap');
+  </style>
```

Comment:
Tip! When you have selected the fonts in Google fonts there is a html link that you can add instead of this :)